docs/js/custom.js

```diff
@@ -202,4 +202,7 @@ $(document).ready(function() {
             closeBtn.closest('.notification').setAttribute('hidden', 'hidden');
         });
     });
+
+    // Mark higher-level nodes with "New" pill, not only the actual item
+    $(".pill.new:not([hidden])").parents(".md-nav__item").uniqueSort().children('label').children(".pill.new[hidden]").removeAttr('hidden');
```

Comment:
I run the test, two new-pilled siblings doesn't bring twice their common parent, `uniqueSort` is useless.
```suggestion
    $(".pill.new:not([hidden])").parents(".md-nav__item").children('label').children(".pill.new[hidden]").removeAttr('hidden');
```

Reply:
You're right, this makes it a bit simpler to understand, thanks!

My initial solution was inserting new elements directly into the DOM, only in the second iteration I've changed to removing the attribute - and indeed it's not needed here